.github/workflows/ci.yml

```diff
@@ -17,40 +17,32 @@ env:
 
 jobs:
     test:
-        name: test ${{ matrix.os }} (${{ matrix.flags.name }})
+        name: test ${{ matrix.flags.name }} (${{ matrix.os }})
         runs-on: ${{ matrix.os }}
         strategy:
             fail-fast: false
             matrix:
-                os: ["ubuntu-latest", "macos-latest"]
+                os: ["ubuntu-latest", "macos-latest", "windows-latest"]
                 flags:
                     - name: no default features
-                      flags: --workspace --no-default-features
+                      flags: --no-default-features
                     - name: default features
-                      flags: --workspace
+                      flags: ""
                     - name: all features
-                      flags: --workspace --all-features
+                      flags: --all-features
```

Comment:
`--workspace` is not needed, because the only crates that are not included are the example ones, which are tested in another job. This allows removing the windows hacks.